Sources/ArcGISToolkit/Components/FeatureFormView/FormInputStyle.swift

```diff
@@ -17,20 +17,29 @@ import SwiftUI
 /// Provides a frame minimum height constraint, padding, background color and rounded corners for a
 /// form input.
 struct FormInputStyle: ViewModifier {
+    let isTappable: Bool
```

Comment:
To add some context to this change:

This type adds a style to each row in the feature form (which is what a `Section` would have done by default).
But I decided to add this here because in a few scenarios the whole row is tappable so this is where the hover effect should be added to so it indicates the row is tappable. 

This is an unusual change because ideally we would have a row and simply putting a button in that row would make the whole row tappable (which is what `Section` did by default).